# TAP results for test_foo.py
ok 1 - test_foo.py::test
ok 2 - test_foo.py::test_foo_3
not ok 3 - test_foo.py::test_foo_4
# 
#     def test_foo_4():
# >       assert 0
# E       assert 0
# 
# test_foo.py:4: AssertionError
not ok 4 - test_foo.py::test_foo_4
# 
#     def test_foo_4():
# >       assert 0
# E       assert 0
# 
# test_foo.py:4: AssertionError
ok 5 - test_foo.py::test_foo_4
ok 6 - test_foo.py::test_foo_3
ok 7 - test_foo.py::test_foo_4
ok 8 - test_foo.py::test_foo
ok 9 - test_foo.py::test
ok 10 - test_foo.py::test_1
ok 11 - test_foo.py::test_foo
ok 12 - test_foo.py::test_that # SKIP condition: hasattr(sys, 'platform')
ok 13 - test_foo.py::test_that # SKIP invalid platform
ok 14 - test_foo.py::test_foobar # SKIP second_condition
ok 15 - test_foo.py::test_foobar
ok 16 - test_foo.py::test_foo[0]
ok 17 - test_foo.py::test_foo[1]
ok 18 - test_foo.py::test_foo[2]
ok 19 - test_foo.py::test_foo[3]
ok 20 - test_foo.py::test_foo[4]
ok 21 - test_foo.py::test_foo[0]
ok 22 - test_foo.py::test_foo[1]
ok 23 - test_foo.py::test_foo[2]
ok 24 - test_foo.py::test_foo[3]
ok 25 - test_foo.py::test_foo[4]
ok 26 - test_foo.py::test_foo[0]
ok 27 - test_foo.py::test_foo[1]
ok 28 - test_foo.py::test_foo[2]
ok 29 - test_foo.py::test_foo[3]
ok 30 - test_foo.py::test_foo[4]
ok 31 - test_foo.py::test_foo[0]
ok 32 - test_foo.py::test_foo[1]
ok 33 - test_foo.py::test_foo[2]
ok 34 - test_foo.py::test_foo[3]
ok 35 - test_foo.py::test_foo[4]
ok 36 - test_foo.py::test_foo[0]
ok 37 - test_foo.py::test_foo[1]
ok 38 - test_foo.py::test_foo[2]
ok 39 - test_foo.py::test_foo[3]
ok 40 - test_foo.py::test_foo[4]
ok 41 - test_foo.py::test_foo[5]
ok 42 - test_foo.py::test_foo[6]
ok 43 - test_foo.py::test_foo[7]
ok 44 - test_foo.py::test_foo[8]
ok 45 - test_foo.py::test_foo[9]
ok 46 - test_foo.py::test_foo[10]
ok 47 - test_foo.py::test_foo[11]
ok 48 - test_foo.py::test_foo[12]
ok 49 - test_foo.py::test_foo[13]
ok 50 - test_foo.py::test_foo[14]
ok 51 - test_foo.py::test_foo[0]
ok 52 - test_foo.py::test_foo[1]
ok 53 - test_foo.py::test_foo[2]
ok 54 - test_foo.py::test_foo[3]
ok 55 - test_foo.py::test_foo[4]
ok 56 - test_foo.py::test_foo[5]
ok 57 - test_foo.py::test_foo[6]
ok 58 - test_foo.py::test_foo[7]
ok 59 - test_foo.py::test_foo[8]
ok 60 - test_foo.py::test_foo[9]
ok 61 - test_foo.py::test_foo[10]
ok 62 - test_foo.py::test_foo[11]
ok 63 - test_foo.py::test_foo[12]
ok 64 - test_foo.py::test_foo[13]
ok 65 - test_foo.py::test_foo[14]
ok 66 - test_foo.py::MyTestCase.test_one # SKIP skipping due to reasons
ok 67 - test_foo.py::MyTestCase.test_should_run
ok 68 - test_foo.py::test_foo # SKIP <Skipped instance>
1..68
